src/rootUtils.h

```diff
@@ -30,6 +30,41 @@ constexpr static auto metaTreeName = "podio_metadata";
  */
 constexpr static auto paramBranchName = "PARAMETERS";
 
+/**
+ * Names of the fields with the keys and values of the generic parameters for
+ * the RNTuples until map types are supported
+ */
+constexpr static auto intKey = "GPIntKeys";
+constexpr static auto floatKey = "GPFloatKeys";
+constexpr static auto doubleKey = "GPDoubleKeys";
+constexpr static auto stringKey = "GPStringKeys";
+
+constexpr static auto intValue = "GPIntValues";
+constexpr static auto floatValue = "GPFloatValues";
+constexpr static auto doubleValue = "GPDoubleValues";
+constexpr static auto stringValue = "GPStringValues";
+
+/**
+ * Name of the field with the list of categories for RNTuples
+ */
+constexpr static auto availableCategories = "availableCategories";
```

Comment:
I suppose this needs to be stored with RNTuples, because there is no easy way (yet) to get the "branches" from the file?

Reply:
Exactly, at least I couldn't find an easy one